website/pages/en/index.js

```diff
@@ -0,0 +1,216 @@
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'top',
+        title: '50+ Songs'
+      },
+      {
+        content: 'Compete with other players online.',
+        image: imgUrl('docusaurus.svg'),
+        imageAlign: 'top',
+        title: 'Online Rankings'
+      }
+    ]}
+  </Block>
+)
+
+const FeatureCallout = props => (
+  <div
+    className='productShowcaseSection highlightBackground paddingBottom'
+    style={{ textAlign: 'center' }}
+  >
+    <h2>Web-based</h2>
+    <MarkdownBlock>{'Play instantly from your browser.<br>No extra plugins required.'}</MarkdownBlock>
+  </div>
+)
+
+const FooterButtons = props => (
+  <div
+    className='productShowcaseSection highlightBackground'
+    style={{ textAlign: 'center' }}
+  >
+    <PromoSection>
+      <Button href='https://bemuse.ninja' target='blank' rel='noopener noreferrer'>Try It Out</Button>
+      <Button href={docUrl('users-gameplay.html', props.language || '')}>Read User Docs</Button>
+      <Button href={docUrl('developers-getting-started.html', props.language || '')}>Read Developer Docs</Button>
+    </PromoSection>
+  </div>
+)
+
+const BMSMode = props => (
+  <Block background='light'>
+    {[
+      {
+        content: `You can play using an [IIDX controller](https://www.youtube.com/watch?v=EOgI37Myqvk) or [MIDI controller](https://www.facebook.com/bemusegame/videos/985712734835136/).`,
+        image: imgUrl('screenshots/gameplay-bms.jpg'),
+        imageAlign: 'right',
+        title: 'BMS Mode'
+      }
+    ]}
+  </Block>
+)
+
+const KeyboardMode = props => (
+  <Block>
+    {[
+      {
+        content: 'Play along the music with your keyboard.',
+        image: imgUrl('screenshots/gameplay-kbd.jpg'),
+        imageAlign: 'left',
+        title: 'Keyboard Mode'
+      }
+    ]}
+  </Block>
+)
+
+const LearnHow = props => (
+  <Block background='light'>
+    {[
+      {
+        content: 'TODO: Custom server setup',
+        image: imgUrl('screenshots/music-select.jpg'),
+        imageAlign: 'right',
+        title: 'Custom Servers'
+      }
+    ]}
+  </Block>
+)
+
+const TryOut = props => (
+  <Block id='try'>
+    {[
+      {
+        content: `Bemuse is free and open source (licensed under [AGPLv3](https://github.com/bemusic/bemuse/blob/master/LICENSE)).\n\nContributions are welcome! Read through our [developer documentation](${docUrl('developers-getting-started.html')}) to get started, and have a look at our [issues page](https://github.com/bemusic/bemuse/issues).`,
+        image: imgUrl('screenshots/oss.png'),
+        imageAlign: 'left',
+        title: 'Open Source'
+      }
+    ]}
+  </Block>
+)
+
+class Index extends React.Component {
+  render () {
+    let language = this.props.language || ''
+
+    return (
+      <div>
+        <HomeSplash language={language} />
+        <div className='mainContainer'>
+          <Features />
+          <FeatureCallout />
+          <BMSMode />
+          <KeyboardMode />
+          <LearnHow />
```

Comment:
I think there is a loss of cohesion here.

If I wanted to add a new row, I needed to create a new component (above) and add it here. So, 2 places to edit. I suggest doing one of these:

- **Option A:** Inline the blocks here.

- **Option B:** Instead of 3 components, use one component (e.g. `FeatureTour`) that renders 3 blocks all at once. (We can make use of `<>` (React.Fragment) component.)

In either case, after the change, we now only need to change 1 place to add a new block.